docs/release.md

```diff
@@ -344,6 +344,36 @@ git push upstream vX.Y.Z
 ./packages/cli/bin/carbon-cli.js changelog vA.B.C..vX.Y.Z
 ```
 
+### Releasing a new major for a single package
+
+In some cases, a single package in the monorepo may need a major version bump.
+This can be done without having to do a major version bump across all packages.
+
+For instance:
+
+- `eslint-config-carbon` needs a new major
+- All other packages should only be bumped as a new patch
+- The tag for the release should remain at the curent major `v11.x`, and not be
+  bumped to `v12.x`
+
+To do this, packages must be versioned manually.
+
+1. Switch to `main`
+1. Pull in latest `git pull upstream main`
+1. Make a new branch with the version you're going to release, e.g.
+   `git checkout -b release/v11.23.1`
+1. Run `yarn lerna version --no-git-tag-version --no-push`
+1. An interactive prompt will be presented - select the appropriate version bump
+   for each package
```

Comment:
Is this the step (in this example) that you would bump `eslint-config-carbon` to `12.x`, while the others are are just bumped up to the next patch version?

Reply:
@tw15egan Correct - it's kind of hard to document, but this is what the output looks like from when I did it today:

```sh
➜  monorepo git:(main) yarn lerna version --no-git-tag-version --no-push
lerna notice cli v6.4.0
lerna info versioning independent
lerna notice FYI git repository validation has been skipped, please ensure your version bumps are correct
lerna info Looking for changed packages since v11.23.0
lerna info ignoring diff in paths matching [
lerna info ignoring diff in paths matching   'actions/**',
lerna info ignoring diff in paths matching   '**/__fixtures__/**',
lerna info ignoring diff in paths matching   '**/__tests__/**',
lerna info ignoring diff in paths matching   '**/docs/**',
lerna info ignoring diff in paths matching   '**/e2e/**',
lerna info ignoring diff in paths matching   '**/examples/**',
lerna info ignoring diff in paths matching   '**/*.md'
lerna info ignoring diff in paths matching ]
lerna WARN version Skipping working tree validation, proceed at your own risk
? Select a new version for eslint-config-carbon (currently 2.20.0) Major (3.0.0)
? Select a new version for jest-config-carbon (currently 1.6.0) Patch (1.6.1)
? Select a new version for codesandbox-styles (currently 0.26.0) Patch (0.26.1)
? Select a new version for codesandbox-with-sass-compilation (currently 0.24.0) Patch (0.24.1)
? Select a new version for codesandbox (currently 0.24.0) Patch (0.24.1)
? Select a new version for custom-theme (currently 0.21.0) Patch (0.21.1)
? Select a new version for incremental-migration (currently 0.23.0) Patch (0.23.1)
? Select a new version for examples-light-dark (currently 0.21.0) Patch (0.21.1)
? Select a new version for examples-nextjs (currently 0.23.0) Patch (0.23.1)
? Select a new version for vite (currently 0.21.0) Patch (0.21.1)
? Select a new version for carbon-components-react (currently 8.23.0) Patch (8.23.1)
? Select a new version for carbon-components (currently 11.23.0) Patch (11.23.1)
? Select a new version for @carbon/elements (currently 11.19.0) Patch (11.19.1)
? Select a new version for @carbon/icons-react (currently 11.16.0) Patch (11.16.1)
? Select a new version for @carbon/icons-vue (currently 10.65.0) Patch (10.65.1)
? Select a new version for @carbon/icons (currently 11.16.0) Patch (11.16.1)
? Select a new version for @carbon/react (currently 1.23.0) Patch (1.23.1)
? Select a new version for @carbon/sketch (currently 11.19.0) Patch (11.19.1)
? Select a new version for @carbon/styles (currently 1.23.0) Patch (1.23.1)
? Select a new version for @carbon/upgrade (currently 11.6.0) Patch (11.6.1)
? Select a new version for www (currently 0.32.0) Patch (0.32.1)

Changes:
 - eslint-config-carbon: 2.20.0 => 3.0.0
 - jest-config-carbon: 1.6.0 => 1.6.1 (private)
 - codesandbox-styles: 0.26.0 => 0.26.1 (private)
 - codesandbox-with-sass-compilation: 0.24.0 => 0.24.1 (private)
 - codesandbox: 0.24.0 => 0.24.1 (private)
 - custom-theme: 0.21.0 => 0.21.1 (private)
 - incremental-migration: 0.23.0 => 0.23.1 (private)
 - examples-light-dark: 0.21.0 => 0.21.1 (private)
 - examples-nextjs: 0.23.0 => 0.23.1 (private)
 - vite: 0.21.0 => 0.21.1 (private)
 - carbon-components-react: 8.23.0 => 8.23.1
 - carbon-components: 11.23.0 => 11.23.1
 - @carbon/elements: 11.19.0 => 11.19.1
 - @carbon/icons-react: 11.16.0 => 11.16.1
 - @carbon/icons-vue: 10.65.0 => 10.65.1
 - @carbon/icons: 11.16.0 => 11.16.1
 - @carbon/react: 1.23.0 => 1.23.1
 - @carbon/sketch: 11.19.0 => 11.19.1 (private)
 - @carbon/styles: 1.23.0 => 1.23.1
 - @carbon/upgrade: 11.6.0 => 11.6.1
 - www: 0.32.0 => 0.32.1 (private)

? Are you sure you want to create these versions? Yes
lerna info execute Skipping git tag/commit
lerna info execute Skipping git push
lerna info execute Skipping releases
lerna success version finished
```

You can see I selected `major` in the prompt for `eslint-config-carbon`, and selected `patch` for all the others. Do you think there's a good way we could document this?

Reply:
This is good; I wanted to confirm if I read it correctly! Nice work 👍🏻